### PR TITLE
feature (AlertService): provide functionality to manage alerts

### PR DESCRIPTION
**Pull request by pweyrich**
@plentymarkets/team-terra

### Definition of Done
Documentation
- [x] Changelog
- [x] Example (updated or created)
- [x] JSDoc

Testing
- [x] Unit Test (updated or created)
- [x] Unit Test successful
- [x] Person 1
- [x] Person 2

Browser-Support
- [x] Chrome
- [x] Firefox
- [x] Safari

### CODE REVIEW

**Comment by zhajjhassan** [comment]
Getestet im Example und in Terra, funktioniert gut.
TsLint Fehler fixen dann gibbes Approve ;)

**Comment by pweyrich** [comment]
@zhajjhassan done

File: src/app/components/alert/example/terra-alert.component.example.html [hunk 1]
```diff
@@ -10,5 +10,8 @@
...
               (click)='showWarningAlert()'>
 </terra-button>
-<terra-alert-panel style="display:none;"></terra-alert-panel>
-<!-- Display:none hier bitte ignorieren -->
\ No newline at end of file
+<terra-button inputCaption="alert via service"
+              (click)='showAlertUsingService()'>
+</terra-button>
+<terra-alert-panel></terra-alert-panel>
+<!-- Display:none hier bitte ignorieren -->
```
Comment (zhajjhassan) [inline src/app/components/alert/example/terra-alert.component.example.html]: kannste auch gleich mitlöschen

**Comment by pweyrich** [comment]
bildschirmfoto 2019-01-14 um 13 30 03
Unit tests successful

**Pull request merged**